.gitignore

```diff
@@ -33,3 +33,5 @@ sysinfo.txt
 # FFG imported content
 /content/D2E/ffg
 /content/MoM/ffg
+# Valkyrie ini files importing ffg ones
```

Comment:
If you merge back from development I have moved these files, this change is not required.